website/src/components/Edit/MetadataForm.tsx

```diff
@@ -278,21 +175,3 @@ export const SubmissionIdRow: FC<SubmissionProps> = ({ submissionId }) => (
         <td className='w-full'>{submissionId}</td>
     </tr>
 );
-
-const mapErrorsAndWarnings = (
```

Comment:
Just as a general remark - when you are just moving functions from one file to another could you put that in a separate PR? This PR has a lot of actual code changes and adding in this refactoring (i.e. moving this function and moving EditableSequences) makes it harder to review.

Reply:
Makes a lot of sense! I will try to do that more.